app/models/challenge_assignment.rb

```diff
@@ -209,16 +203,27 @@ def offering_pseud
   end
 
   def requesting_pseud
-    request_signup ? request_signup.pseud : (pinch_request_signup ? pinch_request_signup.pseud : nil)
+    if request_signup
+      request_signup.pseud
+    else
+      (pinch_request_signup ? pinch_request_signup.pseud : nil)
+    end
   end
 
-
   def offer_byline
-    offer_signup && offer_signup.pseud ? offer_signup.pseud.byline : (pinch_hitter ? (pinch_hitter.byline + "* (pinch hitter)") : "- none -")
+    if offer_signup && offer_signup.pseud
+      offer_signup.pseud.byline
+    else
+      (pinch_hitter ? (pinch_hitter.byline + "* (pinch hitter)") : "- none -")
```

Comment:
Is there a particular reason why None is capitalized request byline but not offer byline?

Reply:
Probably not, the previous code is quite old. From what I could find out from talking to people, this basically never shows up in emails, since the user would have to be deleted for that.